index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inspiration&family=Nanum+Gothic:wght@400;700;800&display=swap"
+    rel="stylesheet">
+
+  <link rel="stylesheet" href="css/style.css">
+  <title>Albert Einstein Tribute Page</title>
+</head>
+
+<body id="home">
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+  <script src="js/index.js"></script>
+  <header id="header" class="navbar">
+    <div>
+      <h1 class="logo"><a href="#home">E = mc2</a></h1>
+    </div>
+    <ul>
+      <li><a href="#Life">Life</a></li>
+      <li><a href="#Career">Career</a></li>
+      <li><a href="#Awards">Awards</a></li>
+    </ul>
+  </header>
+  <main id="main" class="main-container">
+    <h1 id="title" class="title">Dr. Albert Einstein</h1>
+    <h3 id="subtittle" class="subtitle">The father of modern physics.</h3>
+    <div id="img-div" class="img-div">
+      <img
+        src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/3e/Einstein_1921_by_F_Schmutzer_-_restoration.jpg/220px-Einstein_1921_by_F_Schmutzer_-_restoration.jpg"
+        alt="" id="image">
+      <span id="img-caption" class="img-caption">Einstein in 1921, by Ferdinand Schmutzer</span>
+    </div>
+    <div class="content">
+      <p id="tribute-info" class="tribute-info">14 March 1879 - 18 April 1955) was a German-born theoretical physicist,
+        widely acknowledged
+        to be one of the greatest physicists of all time. Einstein is best known for developing the theory of
+        relativity, but he also made important contributions to the development of the theory of quantum mechanics.
+        Relativity and quantum mechanics are together the two pillars of modern physics.</p>
+      <a id="tribute-link" class="btn" href="https://en.wikipedia.org/wiki/Albert_Einstein" target="_blank">Learn More
```

Comment:
**Webhint linter Error:**  _Link 'rel' attribute should include 'noopener'._
Please add `rel="noopener"` inside the `<a>` tag.
`rel=”noopener”` is an HTML attribute that can be added to external links to prevent the opening page to gain any kind of access to the original page.